e2e/rbd.go

```diff
@@ -359,7 +359,10 @@ var _ = Describe("RBD", func() {
 					}
 
 					deleteResource(rbdExamplePath + "storageclass.yaml")
-					createRBDStorageClass(f.ClientSet, f, nil, map[string]string{"csi.storage.k8s.io/fstype": "xfs"})
+					parameters := map[string]string{"mkfsOptions": "-m reflink=0",
```

Comment:
I'd like to recommend not "snuggling" the value on the first line of the multi-line map when possible. I find it very easy to miss important content when some lines are snuggled together but others are not.